docker/nmslib/Dockerfile

```diff
@@ -24,6 +24,7 @@ WORKDIR /home/test
 ADD --chown=test requirements.txt .
 RUN pip3 install -r requirements.txt
 ADD --chown=test okpt okpt
+ADD --chown=test dataset dataset
```

Comment:
Is dataset a mounted volume? What happens on host machine if we change ownership here?

Reply:
I've updated the `docker-compose.yml` to mount `dataset` instead

Reply:
What happens with permissions here?

Reply:
It shouldn't change the permissions on the host machine.